Tests/Marketplace/marketplace_services.py

```diff
@@ -1798,6 +1819,49 @@ def collect_content_items(self):
                             'details': content_item.get('details', ''),
                         })
 
+                    elif current_directory == PackFolders.PARSING_RULES.value:
+                        self.is_siem_pack(content_item, 'ParsingRule')
+                        folder_collected_items.append({
+                            'id': content_item.get('id', ''),
+                            'name': content_item.get('name', ''),
+                        })
+
+                    elif current_directory == PackFolders.MODELING_RULES.value:
+                        self.is_siem_pack(content_item, 'ModelingRule')
+                        folder_collected_items.append({
+                            'id': content_item.get('id', ''),
+                            'name': content_item.get('name', ''),
+                        })
+
+                    elif current_directory == PackFolders.CORRELATION_RULES.value:
+                        self.is_siem_pack(content_item, 'CorrelationRule')
+                        folder_collected_items.append({
+                            'id': content_item.get('global_rule_id', ''),
+                            'name': content_item.get('name', ''),
+                            'description': content_item.get('description', ''),
+                        })
```

Comment:
We don't need to call the `is_siem_pack` on these items as we already know at this point (the same way you added the SIEM_RULES_OBJECTS) that these items indicate a SIEM pack. 
This means you can automatically add the tag.